modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -1,30 +1,38 @@
-= Creating an API instance on API Manager
+= Creating an API instance
```

Comment:
```suggestion
= Creating an API Instance
```

Reply:
Because this topic actually contains the steps, I think it should be titled "Create an API Instance."

Reply:
Done! Thanks!